src/RoslynInsertionTool/scripts/OneOffInsertion.ps1

```diff
@@ -16,7 +16,8 @@ param([string] $clientId,
       [string] $visualStudioBranchName,
       [string] $titlePrefix,
       [string] $writePullRequest,
-      [int] $insertionCount)
+      [int] $insertionCount,
+      [string] $autoComplete)
```

Comment:
Not super familiar with PS but would adding this param break any existing uses of this script or is it optional?

Reply:
I know basically nothing about powershell. I think if you are able to run `.\src\RoslynInsertionTool\scripts\VS-Insertion.ps1` without having to add an argument to OneOffInsertion cmdlet call we are good. @jmarolf?

Reply:
Just tested this out and if no value is passed it defaults autoComplete=false